src/nunit.analyzers.tests/ClassicModelAssertUsage/AreEqualClassicModelAssertUsageCodeFixTests.cs

```diff
@@ -88,6 +120,24 @@ public void TestMethod()
             RoslynAssert.CodeFix(analyzer, fix, expectedDiagnostic, code, fixedCode, fixTitle: ClassicModelAssertUsageCodeFix.TransformToConstraintModelDescription);
         }
 
+        [Test]
+        public void VerifyAreEqualFixWithMessageAnd2ParamsInNonstandardOrder()
+        {
+            var code = TestUtility.WrapMethodInClassNamespaceAndAddUsings(@"
+        public void TestMethod()
+        {
+            var actual = 3d;
+            ↓ClassicAssert.AreEqual(delta: 0.0000001d, expected: 2d, actual: actual, args: new[] { ""Guid.NewGuid()"", Guid.NewGuid().ToString() }, message: ""message-id: {0}, {1}"");
+        }");
+            var fixedCode = TestUtility.WrapMethodInClassNamespaceAndAddUsings(@"
+        public void TestMethod()
+        {
+            var actual = 3d;
+            Assert.That(actual: actual, Is.EqualTo(expected: 2d).Within(0.0000001d), $""message-id: {""Guid.NewGuid()""}, {Guid.NewGuid().ToString() }"");
+        }"); // TODO: Fix trailing trivia if possible
```

Comment:
@manfred-brands Do you think this is worth looking into? I haven't spent too much time on this yet, but this is likely the same problem as https://github.com/nunit/nunit.analyzers/issues/713.

Reply:
Ah, I think this was related https://github.com/nunit/nunit.analyzers/blob/master/src%2Fnunit.analyzers%2FHelpers%2FCodeFixHelper.cs#L44, but I'm not sure if this should be fixed in this PR or file another issue for this bug. I want to avoid a further scope creep.

Reply:
Found the bug; it's simple enough that I'm okay with making it a part of this PR.